<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayDialog</class>
 <widget class="QDialog" name="DisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>343</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>426</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>18</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="allPairsButton">
      <property name="text">
       <string>Display All Pairs</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="showStructuresRadioButton">
      <property name="text">
       <string>Display Only Pairs in Structures</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="showSolutionsRadioButton">
      <property name="text">
       <string>Display Only Pairs in Solutions</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="pairSubsetButton">
      <property name="text">
       <string>Display Pair Subset...</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>112</width>
     <height>77</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="displaySettingsBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>401</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>301</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>9898798</height>
    </size>
   </property>
   <property name="title">
    <string>Pair Subset Settings</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="showHoldCheckBox">
         <property name="text">
          <string>Show Pairs on Hold</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="showIncompatibleCheckBox">
         <property name="text">
          <string>Show Pairs with No Compatibilities</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>218</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="maxPRACheckBox">
         <property name="text">
          <string>Maximum PRA to Display</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="maxPRASpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSlider" name="horizontalSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="minPRASpinBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="minPRACheckBox">
         <property name="text">
          <string>Minimum PRA to Display</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DisplayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DisplayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPRASpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>minPRASpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPRASpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>maxPRASpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPRACheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minPRASpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPRACheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPRACheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxPRASpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxPRACheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontalSlider_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pairSubsetButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>displaySettingsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
